.drone.yml

```diff
@@ -7,6 +7,7 @@ pipeline:
     image: golang:1.8
     commands:
       - go get -u github.com/drone/drone-ui/dist
+      - go get -u github.com/drone/expr
```

Comment:
we can vendor this repository, and then remove this line. Will add a comment with instructions shortly